get-started/training.md

```diff
@@ -1,14 +1,19 @@
 ---
-layout: post
+layout: page
 title: Search.gov Training
 crumbname: Training
 category: manual
 tags: help-manual training videos
+date: June 20, 2019
+last_modified_at: June 20, 2019
 breadcrumbs:
```

Comment:
We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs

Reply:
Going to remove these breadcrumbs since we have the sidenav now.